### PR TITLE
CHANGELOG: Add note on enableGitServerCommandExecFilter

### PR DESCRIPTION
**Pull request by indradhanush**
Blocked by #31318. 

## Test plan

Not applicable. Contains changes to changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ All notable changes to Sourcegraph are documented in this file.
 ### Added
 
 - Code in search results is now selectable (e.g. for copying). Just clicking on the code continues to open the corresponding file as it did before. [#30033](https://github.com/sourcegraph/sourcegraph/pull/30033)
+- Added `experimentalFeatures.enableGitServerCommandExecFilter` (defaults to `true`) to enable a patch for a security issue in gitserver. For more detail see our security advisor [#TBD](insert-link-to-advisory)
```
Comment (indradhanush) [inline CHANGELOG.md:19]: @sourcegraph/security When the security advisory is public, this public link will need to be inserted here.

Comment (david-sandy) [reply CHANGELOG.md]: Done

**Comment by jplahn** [comment]
@indradhanush can we open this again or close it if you think it's too late for the changelog?

**Comment by indradhanush** [comment]
Since the feature flag was added as a potential stop gap to revert the fix if it introduced a regression just in case, and after verifying the feature flag is enabled by default, I am going to close this PR since I don't think we need to add this to the changelog anymore, especially since we don't want anyone setting this flag to false and disabling the patch.

**Pull request closed without merging**